pkgs/development/python-modules/wxpython/4.2.nix

```
@@ -61,6 +67,15 @@ buildPythonPackage rec {
     })
   ];
 
+  postPatch = lib.optionalString (pythonAtLeast "3.12") ''
```

```suggestion
  postPatch = ''
```
Please do this unconditional so that we spot issues easier